internal/commands/scan.go

```diff
@@ -2492,9 +2486,50 @@ func validateCreateScanFlags(cmd *cobra.Command) error {
 		return errors.Errorf("Invalid value for --project-private-package flag. The value must be true or false.")
 	}
 
+	err = validateThresholds(cmd)
+	if err != nil {
+		return err
+	}
+
 	return nil
 }
 
+func validateThresholds(cmd *cobra.Command) error {
+	threshold, _ := cmd.Flags().GetString(commonParams.Threshold)
+	if threshold == "" {
+		return nil
+	}
+	threshold = rearrangeUserInputThresholds(threshold)
```

Comment:
there is a change we are doing the rearrange twice? cause i see it called when starting a scan too

Reply:
No, this line replace the word kicks with iac-security